app/services/build_action_logger.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class BuildActionLogger
+  def initialize(build_action)
+    @build_action = build_action
+  end
+
+  def info(message, context: nil)
```

Comment:
As long as we now need context every time, I think it's better to get rid of default value